agent_shared/src/config.rs

```diff
@@ -25,6 +36,7 @@ pub struct ApplicationConfiguration {
     pub external_server_response_timeout_ms: Option<u64>,
     pub domain_linkage_enabled: bool,
     pub secret_manager: SecretManagerConfig,
+    pub ledger_sponsoring_service: Option<LedgerSponsoringService>,
```

Comment:
We should group all external services under a "external_services" key. Currently, there's just one, but there are more to come. There could also be a flag for the whole section to forbid all external communication or have more general settings that apply for all external services.

Reply:
As discussed, I've now removed any ledger sponsoring functionality from this PR 👍